src/com/trilead/ssh2/transport/KexManager.java

```diff
@@ -215,12 +215,20 @@ NegotiatedParameters mergeKexParameters(KexParameters client, KexParameters serv
 
 		
 
-	
-			np.lang_client_to_server = getFirstMatch(client.languages_client_to_server,
-					server.languages_client_to_server);
-		
-			np.lang_server_to_client = getFirstMatch(client.languages_server_to_client,
-					server.languages_server_to_client);
+			try {
```

Comment:
this revert was much needed to not fail connections for a unneeded values like this. 

see comments on: https://github.com/jenkinsci/trilead-ssh2/pull/222/files/f9db40d327e271ec6bc4c22c9eb79aac25d29ff3